### ISSUE TITLE
it don't work in IE browser, don't support NodeList forEach methods?

### ISSUE THREAD

**Issue by WangXiaoyugg** [akiran/react-slick#1142]
I use the package in project,  when I test the carousel in IE Browser, even in IE 11 , it don't work ,  I see the error tips , I find the IE don't support NodeList.prototype.forEach methods, this is reason cause the Carousel can't
work, I hope the Carousel can use in IE browser

**Comment by imornar** [comment]
Yeah, same issue here, carousel does not work in IE 11

**Comment by laveesingh** [comment]
Which version of react-slick are you using?

**Comment by imornar** [comment]
latest (0.21.0) version having NodeList forEach issue in IE 11.

**Comment by WangXiaoyugg** [comment]
latest verison ,  I have solve the problem , I find a polyfill in mdn , and use the polyfill in my project entry;
```
if (window.NodeList && !NodeList.prototype.forEach) {
    NodeList.prototype.forEach = function (callback, thisArg) {
        thisArg = thisArg || window;
        for (var i = 0; i < this.length; i++) {
            callback.call(thisArg, this[i], i, this);
        }
    };
}
```

**Comment by wszhi** [comment]
Same issue with my carousel. And I add a polyfill too. It works for me.

(function () {
  if (typeof NodeList.prototype.forEach === "function") return false;
  NodeList.prototype.forEach = Array.prototype.forEach;
})();

**Comment by laveesingh** [comment]
This problem was apparently fixed in version 0.22.0. Please make sure you're using the latest version.
